src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/Azure.Core.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <ProjectGuid>{84491222-6C36-4FA7-BBAE-1FA804129151}</ProjectGuid>
+    <LangVersion>7.2</LangVersion>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="nunit" Version="3.10.1" />
+    <PackageReference Include="nunit" Version="3.10.1" />
```

Comment:
Why is nunit twice?

Reply:
Copy/paster error -- will fix.

Reply:
Is there a reason this project is using nunit as opposed to the standard xunit?

Reply:
I have filed https://github.com/Azure/azure-sdk-for-net/issues/5221 to track the move to XUnit